Remotion/Web/Core/ExecutionEngine/UrlMapping/UrlMappingConfiguration.cs

```diff
@@ -43,7 +43,10 @@ public class UrlMappingConfiguration: ConfigurationBase
 
   public static UrlMappingConfiguration CreateUrlMappingConfiguration (string configurationFile)
```

Comment:
what do we need this method for anylonger (did not check usages myself, just seems redundant now?)

The "one" usage downstream goes away so we can mark the method as obsolete and document how to manually do it. In general, we expect projects to switch to the new infrastructure.

Reply:
28 usages in the tests...

Reply:
Move to tests?